dashboard/test/models/ability_test.rb

```diff
@@ -83,6 +87,8 @@ class AbilityTest < ActiveSupport::TestCase
     assert !ability.can?(:destroy, Level)
     assert !ability.can?(:destroy, Activity)
 
+    assert !ability.can?(:read, Section)
```

Comment:
I was wondering the same thing.  I'd also prefer `cannot?` but I also like to be minimal around changes like this, especially where it's my first time touching it, so kinda torn.